docker/api/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gorghoa/php7
```

Comment:
You should better use [API Platform Dockerfile](https://github.com/api-platform/api-platform/blob/master/Dockerfile)